MonMaraicher/Features/Map/MapViewModel.swift

```diff
@@ -195,16 +195,20 @@ extension MapViewModel {
 
     struct Marker: Identifiable, Hashable {
 
-        let id: Int
+        let id: UUID
         let title: String
         let address: Address
         let farmer: Farmer
 
         init(farmer: Farmer, address: Address) {
-            self.id = farmer.id
+            self.id = UUID()
```

Comment:
Since your id is unique because you create an UUID(), you don't have to override the equatable. You have to choose:
- you are using farmer id + override equatable to make id + address of a marker unique
- you are using UUID init which will produce an unique ID